.mergify.yml

```diff
@@ -1,16 +1,18 @@
 ---
 defaults:
   actions:
+  # mergify.io has removed bot_account from its free open source plan
     comment:
-      bot_account: ceph-csi-bot
+      # bot_account: ceph-csi-bot # mergify[bot] will be commenting.
     merge:
-      bot_account: ceph-csi-bot
+      # bot_account: ceph-csi-bot # mergify[bot] will be merging pull requests.
       method: rebase
       rebase_fallback: merge
       strict: smart
-      strict_method: rebase
+      strict_method: rebase # same as rebase action.
     rebase:
-      bot_account: ceph-csi-bot
+      # bot_account: ceph-csi-bot # mergify will randomly pick and use credentials of users
```

Comment:
`bot_account` is deprecated correct? in that case, do we need it commented or we can remove it?

Reply:
`bot_account` is replaced `merge_bot_account` and `update_bot_account`(auto pr rebase) in merge actions and is moved to paid `Premium feature plan`. For comment and rebase, it stays the same but also moved to paid `Premium feature plan`.

Updated pr with those changes.